### PR TITLE
Design System spike

### PR DESCRIPTION
**Pull request by holloway**
The updated [`README.md` is  easier to read here](https://github.com/springload/frontend-starter-kit/tree/spike/design-system/project-template)

### CODE REVIEW

File: webpack-build/src/Global/_typography.scss [hunk 1]
```diff
@@ -0,0 +1,29 @@
+body {
+  // UPDATE_ME
+  color: var(--color-grey-90);
+  font-family: $font-family-sans;
+  font-size: rem(18);
+  line-height: 1.6;
```
Comment (liamjohnston) [inline webpack-build/src/Global/_typography.scss:6]: Tempted to default to using this:
https://twitter.com/hihayk/status/1280979972258172928?s=19

But can do that later.

File: webpack-build/webpack.config.js [hunk 2]
```diff
@@ -100,4 +137,8 @@ module.exports = (env, options) => {
       ],
     },
   };
+
+  console.log(webpackConfig);
```
Comment (liamjohnston) [inline webpack-build/webpack.config.js:141]: assume this is tremporary

**Review by liamjohnston — APPROVED** [review]
Soooo... video is an interesting example. It's not a JS component in Massey - I'm just using a disclosure widget. 

Fair enough to make a react component for the template (as the default standard template format, as discussed). But I think by putting it in `ComponentsInit` is implying that it's a JS component, which it isn't. Thoughts?

**Comment by holloway** [comment]
@liamjohnston yeah Video is incidental and it probably should change.

I think what we want to start with is several examples of progressive enhancement from Wagtail/SS (scraping the DOM for vars), and also just React templating that doesn't care about that

**Pull request merged**